judge/views/contests.py

```diff
@@ -102,7 +98,7 @@ def get_context_data(self, **kwargs):
         context['active_participations'] = active
         context['current_contests'] = present
         context['future_contests'] = future
-        context['now'] = self._now
+        context['now'] = timezone.now()
```

Comment:
I don't like this. We should pull the now from the queryset and only use `timezone.now()` as a last resort.

Reply:
The only use of this `now` is for the contest calendar (only the month and the year). I'm not sure how to pull from the queryset for the calendar.

Reply:
If you follow used `timezone.now()` you could just pull it out of the queryset. If not, you can set a variable to `timezone.now()` and then overwrite in the loop above.